components/sysutils/coreutils/Makefile

```diff
@@ -46,21 +52,25 @@ PATCH_LEVEL=		0
 # Prevent gcc register spill error on 32-bit SPARC
 gcc_OPT.sparc.32 =	-O1
 
-CONFIGURE_PREFIX	 =	/usr/gnu
-CONFIGURE_OPTIONS	+=	--libdir=/usr/lib
+# Recreate configure and Makefile to pick up changes in .../src/local.mk
+COMPONENT_PREP_ACTION = (cd $(@D) ; autoreconf -f && automake)
+
+CONFIGURE_PREFIX	 =	$(GNUDIR)
+CONFIGURE_BINDIR.64	 =	$(GNUBIN)
+CONFIGURE_OPTIONS	+=	CPPFLAGS=" -I$(USRINCDIR)/gmp"
+CONFIGURE_OPTIONS	+=	--libdir=$(USRLIBDIR.$(BITS))
```

Comment:
The path was changed in the manifest in the original recipe. I added the definition of libexecdir now. Do you want the lib in usr/lib/$(MACH64)/coreutils/libstdbuf.so or usr/lib/$(MACH64)/libstdbuf.so?
I do not know other consumers than stdbuf in coreutils, maybe worth asking around.
```
> ldd /usr/lib/* 2> /dev/null | grep libstdbuf
/usr/lib/libstdbuf.so:
> ldd /usr/bin/* 2> /dev/null | grep libstdbuf
>
```